packages/manager/apps/vrack-services/src/public/translations/vrack-services/dashboard/Messages_fr_FR.json

```
@@ -34,5 +34,13 @@
   "vrackServicesInErrorMessage": "vRack Services {{displayName}} est en erreur",
   "subnetCreationOnGoing": " : Sous-réseau {{name}} en cours de création",
   "endpointCreationOnGoing": " : Service endpoint en cours de création",
-  "vrackActionDissociate": "Dissocier"
+  "vrackActionDissociate": "Dissocier",
+  "vrackActionAssociateToAnother": "Associer à un autre vRack",
+  "modalAssociateAnotherVrackTitle": "Associer à un autre vRack à vRack Services",
```

Not sure what's the good fix but currently the sentence doesn't make sense

```suggestion
  "modalAssociateAnotherVrackTitle": "Associer un autre vRack à vRack Services",
```